### PR TITLE
[RA1 Ch2, Ch4] T-shirt removal

### LINKED ISSUE #2038: [RA1 Ch2, Ch4] T-shirt removal
The T-shirt terminology is not used anymore and has been removed from the rest of the documentation.

### PR DESCRIPTION
**Pull request by EditKoselak**
Closes #2038

### CODE REVIEW

File: doc/ref_arch/openstack/chapters/chapter02.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ This chapter includes both "Requirements" that must be satisifed in an RA-1 conf
 ## 2.2 Reference Model Requirements
 
 
-The tables below contains the requirements from the Reference Model to cover the Basic and Network Intensive profiles. 
+The tables below contains the requirements from the Reference Model to cover the Basic and Network Intensive profiles.
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter02.md:49]:
```suggestion
The tables below contain the requirements from the Reference Model to cover the Basic and Network Intensive profiles.
```

**Review by iangardner22 — APPROVED** [review]

**Review by petorre — APPROVED** [review]

**Review by karinesevilla — COMMENTED** [review]
@EditKoselak The numbering of tables in RM has changed. The table for pre-defined compute flavors is 4.13. Can you modify in 4.4.1?

**Review by walterkozlowski — APPROVED** [review]

**Review by ulikleber — APPROVED** [review]
Approved, but please fix the typo raised by pgoyal.

**Comment by EditKoselak** [comment]
> @EditKoselak The numbering of tables in RM has changed. The table for pre-defined compute flavors is 4.13. Can you modify in 4.4.1?

Thanks, done:)

**Review by pgoyal01 — APPROVED** [review]

**Pull request merged**